gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -5,222 +5,395 @@ option java_multiple_files = false;
 option java_package = "io.zeebe.gateway.protocol";
 option go_package = "pb";
 
-message TopologyRequest {
-}
-
-message Partition {
-  int32 partitionId = 1;
-  enum PartitionBrokerRole {
-    LEADER = 0;
-    FOLLOWER = 1;
-  }
-  PartitionBrokerRole role = 3;
-}
+// For a more complete documentation, refer to Zeebe documentation at:
+// https://docs.zeebe.io/grpc/reference.html
 
-message BrokerInfo {
-  int32 nodeId = 1;
-  string host = 2;
-  int32 port = 3;
-  repeated Partition partitions = 4;
-}
-
-message TopologyResponse {
-  repeated BrokerInfo brokers = 1;
-  int32 clusterSize = 2;
-  int32 partitionsCount = 3;
-  int32 replicationFactor = 4;
-}
-
-message WorkflowRequestObject {
-  string name = 1;
-  enum ResourceType {
-    // FILE type means the gateway will try to detect the resource type using the file extension of the name
-    FILE = 0;
-    BPMN = 1;
-    YAML = 2;
-  }
-  ResourceType type = 2;
-  bytes definition = 3;
-}
+// ActiveJobs
```

Comment:
These kind of comments are really verbose/redundant :sweat_smile:

Reply:
It was mostly to group things together, I think maybe we should think about splitting into multiple files and having the `gateway.proto` be simply the service definition? But I can remove them now, they aren't useful.